fixtures/ext-types/lib-with-seperate-swift-modules/Cargo.toml

```diff
@@ -0,0 +1,41 @@
+[package]
+name = "uniffi-fixture-ext-types-swift-seperate-modules"
+edition = "2021"
+version = "0.21.0"
+authors = ["Firefox Sync Team <sync-team@mozilla.com>"]
+license = "MPL-2.0"
+publish = false
+
+[package.metadata.uniffi.testing]
+external-crates = [
+    "uniffi-fixture-ext-types-guid",
+    "uniffi-fixture-ext-types-lib-one",
+    "uniffi-example-custom-types",
+]
+swift-use-seperate-modules = true
```

Comment:
Is this really a testing attribute? I overlooked this expecting to find it in uniffi.toml. How does a "real" consumer specify this in a non-testing context? (This again just shows I'm missing something obvious about this use-case!)

Reply:
Yes this is testing only.  In the real consumer situation, UniFFI will generate a set of `.swift` files and it's up to you to decide how to compile them.  You can either compile them as 1 module or as separate modules.  I wonder if UniFFI should be less hands-off about this, but don't know what changes to suggest.